src/FetchWorkerTool.worker.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-env worker */
+
+let jobsActive = 0;
+const complete = [];
+
+let intervalId = null;
+
+/**
+ * Register a step function.
+ *
+ * Step checks if there are completed jobs and if there are sends them to the
+ * parent. Then it checks the jobs count. If there are no further jobs, clear
+ * the step.
+ */
+const registerStep = function () {
+    intervalId = setInterval(() => {
+        if (complete.length) {
+            postMessage(complete.slice(), complete.map(response => response.buffer).filter(Boolean));
```

Comment:
Could you please add a comment explaining the deal with transferring the buffer to the parent here?